Buildings/Controls/OBC/CDL/Logical/VariablePulse.mo

```diff
@@ -0,0 +1,340 @@
+within Buildings.Controls.OBC.CDL.Logical;
+block VariablePulse
+  "Generate boolean pulse with the width specified by input"
+
+  parameter Real period(unit="s")
+    "Time for one pulse period";
+  parameter Real chaWidThr=0.01
+    "Minimum input change to re-trigger an update of the output interval. It is the ratio of the value change to the original value"
+    annotation (Dialog(tab="Advanced"));
+  parameter Real minTruFalHol(
+    final quantity="Time",
+    final unit="s",
+    final min=Constants.small)=1
+    "Minimum time to hold true or false";
+  parameter Real zerWidThr=0.01
```

Comment:
Why not using `minTruFalHol` to set the bound on `u` below which it is considered always off, or always on. E.g., if `u < uMin` with `uMin = minTruFalHol/period` then `y=false` always, wheree `minTruFalHod` cannot be longer than `period/2`.

Reply:
changed to use the `uMin` and set it as suggested. It is then added to the protected section.